pkg/btf/validation.go

```diff
@@ -295,10 +295,6 @@ spec:
 	for _, key := range syscallinfo.SyscallsNames() {
 		var fn *btf.Func
 
-		if key == "" {
-			continue
-		}
-
```

Comment:
Doesn't this change the behaviour of the loop?

Reply:
It was actually a fix for a bug because I think the author did not understand why they were empty keys in the slice. See this:
https://github.com/cilium/tetragon/pull/983/commits/f021b5de1998c6af6a74e588b882e5780eae0502#diff-468002e48832ac8408184ef20e818fa41c0f35b2a7562d8bc20b902182860bfdL47-R47.

See this commit message https://github.com/cilium/tetragon/pull/983/commits/f021b5de1998c6af6a74e588b882e5780eae0502.

Reply:
Gotcha.

Reply:
Can we change the 'continue' to 'return error' instead? (As per my comment on #1832?